pyflakes/test/test_doctests.py

```diff
@@ -367,12 +367,7 @@ class TestOther(_DoctestMixin, TestOther):
 
 
 class TestImports(_DoctestMixin, TestImports):
-
-    def test_futureImport(self):
-        """XXX This test can't work in a doctest"""
-
-    def test_futureImportUsed(self):
-        """XXX This test can't work in a doctest"""
+    pass
```

Comment:
Ah, of course. Perhaps instead of `pass` we can have a short docstring saying just that?